### ISSUE TITLE
sourcecode line coverage not corret

### ISSUE THREAD

**Issue by FishBlanche** [istanbuljs/babel-plugin-istanbul#284]
I can not understand the coverage report.<br>
why is that line 8 to 11 and line 20 to 22 are covered  but line 12 to line 19 not covered?

![123](https://github.com/istanbuljs/babel-plugin-istanbul/assets/19261291/49bac616-488a-4bbe-ab86-50620d93fa97)

**Comment by zodman** [comment]
same problem here ...

**Comment by Zengzzhao** [comment]
Exactly. I've also noticed that the source code line/column numbers recorded in the window.__coverage__'s data don't match my actual source code.

**Comment by Zengzzhao** [comment]
> Exactly. I've also noticed that the source code line/column numbers recorded in the window.**coverage**'s data don't match my actual source code.没错。我也注意到，** 代码**覆盖率窗口中记录的源代码行号/列号与我的实际源代码不符。

<img width="2300" height="966" alt="Image" src="https://github.com/user-attachments/assets/255009e8-271d-497a-be16-8a77d31560b6" />

@zodman @FishBlanche this is my issue
